client/my-sites/checkout/src/components/checkout-main-content.tsx

```diff
@@ -144,6 +149,27 @@ const SideBarLoadingCopy = styled.p`
 	animation: ${ pulse } 2s ease-in-out infinite;
 `;
 
+const ContactDetailsFormDescription = styled.p`
+	font-size: 14px;
+	color: ${ ( props ) => props.theme.colors.textColor };
+	margin: 0 0 16px;
+`;
+
+function ConditionalContactDetailsMessage( {
```

Comment:
```suggestion
function DomainContactDetailsMessage( {
```
What do you think of this suggestion? Since this element is only shown when a domain is in the cart.